### DISCUSSION TITLE
Disabling auto-complete ?

### DISCUSSION THREAD

**Discussion by xrival** [pulsar-edit/.github#151] [Core Application & Packages]
Hi,
I am new to Pulsar (though I was using Atom before).
While typesetting Latex, if find the auto-complete menu distracting. How to turn this off ?
I searched the settings without success so far. Any idea ? Thanks!

**Comment by confused-Techie** [comment]
@xrival If you'd like to disable autocomplete suggestions from individual packages you can always disable a package that provides them, such as `autocomplete-css` and `autocomplete-html` (Which are the two included by default) but you'd still receive some.

So if you'd like to totally disable all autocomplete suggestions you'd want to toggle off `Show Suggestions On Keystroke` which is a setting of the `autocomplete-plus` package (which powers all autocompletions).

If you're on the latest release of Pulsar you can go to the "Search" on the sidebar of settings and search with the following string `autocomplete-plus enableAutoActivation` then you should see "Show Suggestions On Keystroke" as the first or second option. Clicking it will take you to the proper page in settings to disable it.

**Reply by xrival** [reply]
Thanks a lot! I actually had autocomplete-plus installed, and indeed could solve the problem there.
